### ISSUE TITLE
Action: VeSync: Update devices - nothing happens

### ISSUE THREAD

**Issue by Phoenix-DH** [haext/custom_vesync#26]
### The problem

When using the action in the dev tools, nothing happens.
No report to the log.

### What version of this integration has the issue?

9e95d6d 

### What version of Home Assistant Core has the issue?

2024-11-04

### Diagnostics

---

### Home Assistant log

Nothing is inside the log.

**Comment by gdgib** [comment]
@Phoenix-DH do you have any reason to expect something to happen?  If there are no new, supported devices, then nothing happening would be entirely normal and correct.  Can you provide any reason to believe that's not the case in this situation?

**Comment by Phoenix-DH** [comment]
Hey,
yes, I expect that my Levoit OmniMist 1000s will get grapped.

**Comment by gdgib** [comment]
Honestly, I wonder if we even support that model.  Do you know?  I'm still getting up to speed on being a proper maintainer for this code base.

FWIW, we're probably going to deprecate that action in 1.4.0, since it shouldn't be necessary anymore.

**Comment by Phoenix-DH** [comment]
Hey,

this is the same model thank talking about here:
https://github.com/haext/custom_vesync/issues/20

I am wondering that the thread creator has the modell available or not in HA, I do not.

**Comment by gdgib** [comment]
Good catch! I didn't notice that. Thank you. #20 is the tight issue for future work on this, then.
